GitUI/CommandsDialogs/RevisionDiffControl.cs

```diff
@@ -212,16 +212,14 @@ private string DescribeRevision([CanBeNull] ObjectId objectId, int maxLength = 0
         /// <returns>A description of the selected parent</returns>
         private string DescribeSelectedParentRevision()
         {
-            var parents = DiffFiles.SelectedItemParents
-                .Distinct()
```

Comment:
You added the `.Distinct()` in 1dc1f818932aea85cb9156be8bf5e16e944d69b7 -- for a reason?

Reply:
.Distinct() added to SelectedItemParents in 8923cac336dd14bebbe3bc3da288646ea96b8a6a so this no longer needed here
Two no optimal changes, third time's a charm?